protocol/avdl/chat1/notify.avdl

```diff
@@ -82,6 +83,17 @@ protocol NotifyChat {
     union { null, InboxUIItem } conv;
   }
 
+  record ReactionDelete {
+    string reactionKey;
```

Comment:
@buoyad any other info/changes that would be good to put here?

Reply:
nope, we can look it up easily given this :+1: